api/v1alpha1/openstack_ansibleee_types.go

```diff
@@ -90,7 +90,7 @@ type OpenStackAnsibleEESpec struct {
 	// +kubebuilder:default:=6
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:number"}
 	BackoffLimit *int32 `json:"backoffLimit,omitempty"`
-	// Role is the description of an Ansible Role
+	// Role is a structured Ansible Play
```

Comment:
This is exactly why I would get rid of Role. See my comment on https://github.com/openstack-k8s-operators/openstack-ansibleee-operator/pull/202

Role is a structured play, that only lets you use roles. This is a not needed abstraction, and does not align with any native ansible interface.

Instead, let's just have:

Playbook: use a playbook name from the runner image
Play: type your own playbook play

That is simpler, and more powerful IMO.